docs/what_is_securedrop.rst

```diff
@@ -3,9 +3,121 @@ What Is SecureDrop?
 
 SecureDrop is an open-source whistleblower submission system that media
 organizations can use to securely accept documents from and communicate with
-anonymous sources. It was originally created by the late Aaron Swartz and is
-currently managed by `Freedom of the Press Foundation
-<https://freedom.press>`__.
+anonymous sources.
+
+Purpose
+-------
+
+In many of the recent leak prosecutions in the United States, sources have been
+investigated because authorities are able to retrieve both metadata and content
+of communications from third parties like email and phone providers in secret. 
+SecureDrop attempts to completely eliminate third parties from the equation so
+that news organizations can challenge any legal orders before handing over any 
+data.
+
+SecureDrop also substantially limits the metadata trail that may exist from 
+journalist-source communications in the first place. In addition, it attempts
+to provide a safer environment for those communications than regular corporate
+news networks, which may be compromised.
+
+Project History
```

Comment:
I would suggest we keep the "How it works" pretty high up, maybe just below the "Purpose" section.